Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageTest.cs

```diff
@@ -301,19 +313,37 @@ public void ClearPendingState()
         [TestMethod]
         public void FailToGetALog()
         {
-            var invalidLogEntry = new LogEntry { Channel = StorageTestChannelName, Log = "good luck deserializing me!" };
-            using (var connection = new SQLiteConnection("Microsoft.AppCenter.Storage"))
+            // Prepare data.
+            StorageAdapter adapter = new StorageAdapter();
+            var database = "test.db";
```

Comment:
Why not use `DatabasePath` property? The reason we used GUID as database name is to avoid potential cleanup issues.